types/test/bodybuilder-tests.ts

```diff
@@ -1,6 +1,4 @@
-/// <reference path="../bodybuilder.d.ts" />
-
-import bodybuilder = require('bodybuilder');
+import bodybuilder from 'bodybuilder';
```

Comment:
If we have to change this, it's a breaking change to the types (see #197). Technically currently both ways of importing work, I think.

I cannot think of a way how both ways of importing will keep on working.